ts/webui/src/components/managementExp/NameColumn.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+
+interface NameColumnProps {
+    port: number;
+    expName: string;
+}
+
+class NameColumn extends React.Component<NameColumnProps, {}> {
+    constructor(props: NameColumnProps) {
+        super(props);
+    }
+
+    render(): React.ReactNode {
+        const { port, expName } = this.props;
+        const hostname = window.location.hostname;
+        const protocol = window.location.protocol;
+        const webuiPortal = `${protocol}//${hostname}:${port}/oview`;
+        return (
+            <div className='succeed-padding ellipsis'>
+                <a href={webuiPortal} className='link' target='_blank' rel='noopener noreferrer'>
```

Comment:
Stopped experiments should not be clickable.

And @J-shang , stopped experiments should not have port in `.experiment`.

Reply:
by design, i will discuss with Xingyu

Reply:
now, when cursor hover on the trial id, font change `blue with underline`